src/app/components/article/ArticleForm.js

```diff
@@ -0,0 +1,327 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import Slideout from '../cds/slideout/Slideout';
+import ButtonMain from '../cds/buttons-checkboxes-chips/ButtonMain';
+import TagList from '../cds/menus-lists-dialogs/TagList';
+import TextArea from '../cds/inputs/TextArea';
+import TextField from '../cds/inputs/TextField';
+import LanguagePickerSelect from '../cds/inputs/LanguagePickerSelect';
+import LimitedTextArea from '../layout/inputs/LimitedTextArea';
+import DeleteIcon from '../../icons/delete.svg';
+import inputStyles from '../../styles/css/inputs.module.css';
+import { safelyParseJSON, truncateLength } from '../../helpers';
+import styles from './ArticleForm.module.css';
+
+const ArticleForm = ({
+  handleSave,
+  onClose,
+  handleBlur,
+  handleDelete,
+  articleType,
+  mode,
+  article,
+  team,
+  projectMedia,
+}) => {
+  const title = <FormattedMessage id="article-form-slideout-title" defaultMessage="{mode} {formType}" description="Title for the slideout article form" values={{ mode: mode === 'create' ? 'Create New' : 'Edit ', formType: articleType === 'explainer' ? 'Explainer' : 'Claim & Fact Check' }} />;
```

Comment:
Please refactor this as 4 different full sentences for adequate translation work. In the current implementation there's nothing really to be translated as the contents of `defaultMessage` are placeholder vriables only and the actual content is not translatable in the hardcoded `values` object.